flake.nix

```
@@ -59,6 +60,8 @@
             ];
           vendorHash = "sha256-obtcJTg7V4ij3fGVmZMD7QQwKJX6K5PPslpM1XKCk9Q="; # SHA based on vendoring go.mod
         };
+
+        default = self.packages.${pkgs.system}.tsidp;
       });
 
```

```suggestion

      overlays.default = final: prev: {
        tsidp = self.packages.${final.system}.tsidp;
      };
```
We could provide an overlay in the flake so someone could use this tsidp package instead of the nixpkgs one like this:
```nix
  nixpkgs.overlays = [
    inputs.tsidp-flake.overlays.default 
  ];
```

---

But yeah probably setting the local package as the default is correct. I was also testing with the nixpkgs version so I missed that, sorry!